src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodesDefinition.xml

```diff
@@ -106,6 +106,9 @@
       <Filter Property="IsSystemObject" Value="0" Type="bool" />
       <Filter Property="IsDroppedLedgerView" Value="0" Type="bool" ValidFor="Sql2022|AzureV12"/>
     </Filters>
+    <Properties>
+      <Property Name="LedgerViewType" ValidFor="Sql2022|AzureV12"/>
+    </Properties>
```

Comment:
why can't OE pick up the property list from here instead of requiring the C# code override?